backend/initialize.py

```diff
@@ -0,0 +1,139 @@
+from datetime import datetime
+
+from backend import db
+from backend.models import User, Project
+
+users = [
+    {
+        'public_id': '705897b6-5478-4ba2-822a-c5e5833d4124',
+        'email': 'sander@vandoorn.nl',
+        'username': 'admin',
+        'admin': True,
+        # password: OneTwoThree123
+        'password_hash': 'sha256$N34hfMMN$a6bec8f319443512b9c0cf5a400c66933ab563a5d2ff5d34fd87dbc19f27d3f7',
+        'biography': None,
+        'flag_count': 0,
+    },
+    {
+        'public_id': 'b9229d9b-cb28-4e12-baea-ead89ef0a494',
+        'email': 'melle@dijkstra.nl',
+        'username': 'melle',
+        'admin': False,
+        # password: 123456
+        'password_hash': 'sha256$xjKXjf0B$3efddd4395a0320e0ba6b6fddf14f793fe9dbe0170a7cdc657addda822dd65fd',
+        'biography': 'Ich bin melle, das kleine Krokodil\n' +
+                     'Komm aus Ägypten, das liegt direkt am Nil\n' +
+                     'Zuerst lag ich in einem Ei\n' +
+                     'Dann schni-,schna-,schnappte ich mich frei\n\n' +
+                     'Mi Ma Melle\n' +
+                     'Mima Mima Mell\n' +
+                     'Mi Ma Melle\n' +
+                     'Mima Mima Mell',
+        'flag_count': 20,  # bad boy ;)
+    },
+]
+
+projects = [
+    {
+        'title': 'Building Houses',
+        'short_description': 'We are going to build houses for people in Uganda',
+        'project_plan': 'Our plan is to build cheap, but properly sized houses for people in Uganda.',
+        'owner': users[0]['public_id'],
+        'target_budget': 10000,
+        'current_budget': 4576,
+        'donators': 18,
+        'start_date': datetime.fromtimestamp(1527764457),
+        'end_date': datetime.fromtimestamp(1527766457),
+        'latitude': 0.99190516,
+        'longitude': -58.86762519,
+        # 'collaborators': [
+        #     1, 2
+        # ],
+        'cover': None,
+        # 'country': 'Uganda',
```

Comment:
Waarom is land commented? Mag gewoon active, ik moet create project form er nog wel even op uitbreiden